x/concentrated-liquidity/fees_test.go

```diff
@@ -452,7 +452,7 @@ func (suite *KeeperTestSuite) TestGetInitialFeeGrowthOutsideForTick() {
 		validPoolId = 1
 	)
 
-	initialPoolTickInt, err := math.PriceToTick(DefaultAmt1.ToDec().Quo(DefaultAmt0.ToDec()), DefaultTickSpacing)
+	initialPoolTickInt, err := math.PriceToTickRoundBankers(DefaultAmt1.ToDec().Quo(DefaultAmt0.ToDec()), DefaultTickSpacing)
```

Comment:
can we also add a comment why we use round  bankers here please? Would be super helpful in understanding context